org.eclipse.jdt.ls.core/plugin.xml

```diff
@@ -2,6 +2,7 @@
 <?eclipse version="3.4"?>
 <plugin>
    <extension-point id="org.eclipse.jdt.ls.core.delegateCommandHandler" name="delegateCommandHandler" schema="schema/org.eclipse.jdt.ls.core.delegateCommandHandler.exsd"/>
+   <extension-point id="importers" name="JDT LS Project Importer" schema="schema/importers.exsd"/>
```

Comment:
org.eclipse.jdt.ls.core.importers would be consistent with the other extension points